torch/jit/_monkeytype_config.py

```diff
@@ -124,3 +127,29 @@ def __init__(self):
             pass
 
     monkeytype_trace = None  # noqa: F811
+
+def jit_code_filter(code: CodeType) -> bool:
+    """
+    Custom CodeFilter for Torchscript to trace forward calls
```

Comment:
Can you add some comment about why `forward` needs to be skipped? I don't think I fully understand the reason yet.